src/components/AddPlayerView.js

```diff
@@ -46,16 +47,6 @@ class AddPlayerView extends Component {
         })
     }
 
-    //without standards of v4
-    myUuidGenerator() {
-        function s4() {
-            return Math.floor((1 + Math.random()) * 0x10000)
-                .toString(16)
-                .substring(1);
-        }
-        return s4() + s4() + '-' + s4() + '-' + s4() + '-' +
-            s4() + '-' + s4() + s4() + s4();
-    }
 
     onSubmit(e) {
         e.preventDefault();
```

Comment:
e.preventDefault() old technique there are some new techniques to dealt with this issue consider them.

Reply:
I have researched and haven't came across any alternative to preventDefault(). stopPropagation() is another method but its of no use in this scenario as I don't have to keep it from propagating any parent objects as it spreads out in parent's direction.
We can use return false, which is somewhat a combination of both preventDefault and stopPropagation and returns as soon as its met, but as I don't need to stop propagation, I'm not using return false as well.